Lib/test/test_sqlite3/test_regression.py

```diff
@@ -45,28 +45,24 @@ def test_pragma_user_version(self):
 
     def test_pragma_schema_version(self):
         # This still crashed pysqlite <= 2.2.1
-        con = sqlite.connect(":memory:", detect_types=sqlite.PARSE_COLNAMES)
-        try:
+        with memory_database(detect_types=sqlite.PARSE_COLNAMES) as con:
             cur = self.con.cursor()
```

Comment:
`cur` is not closed now, I think that `con.close()` will close it, just want to double check.

Reply:
Yeah, the connection object implicitly closes all open cursors.